MarkBind/markbind
# Add conditional panel closing transition based on height

**What is the purpose of this pull request?**



Resolves #1836 

**Overview of changes:**
Only activate closing transition for panels when the height is below a certain height.
![Kapture 2022-07-17 at 10 12 17](https://user-images.githubusercontent.com/62977276/179381108-49e2ec23-2e9e-42f9-83ed-86d58dcc4d93.gif)

**Anything you'd like to highlight/discuss:**
Only adjusted closing transitions, opening transition remains the same.
Fixed height is 300 for now, can be adjusted.

**Testing instructions:**
Create a test site and put chunkier placeholder text into the panel to see the difference in closing transitions.



**Proposed commit message: (wrap lines at 72 characters)**
Add conditional panel closing transition based on height



---

**Checklist:** :ballot_box_with_check:
